### PR TITLE
Internal disease links in disease table on diagnosis page now opens in new tab

### LINKED ISSUE #4494: Open link to OMIM/ORPHA from Diagnosis page on new tab
Let's `target='_blank'` it.

### PR DESCRIPTION
**Pull request by TereseBo**
This PR adds a functionality or fixes a bug.

The links for OMIM/ORPHA disease pages which are present in the diagnoses table on the diagnoses page now opens in a new tab.

This PR closes #4494

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1.Verify the relevant links now open in a new tab

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by TB

### CODE REVIEW

**Comment by TereseBo** [comment]
On this branch:
image
 When clicked, the link opens in a new tab

**Comment by northwestwitch** [comment]
Do not merge this one before we merge #4499!

File: scout/server/blueprints/diagnoses/templates/diagnoses/diagnoses.html
Comment (dnil) [reply scout/server/blueprints/diagnoses/templates/diagnoses/diagnoses.html:85]: Oh. I see. Its like a kind of placeholder for the js... 😸

**Review by dnil — APPROVED** [review]
Thank you - now they are consistent! But let's wait with merging this to `main` until we are sure of the patch release content.

**Comment by northwestwitch** [comment]
This is fine to merge now!

**Pull request merged**
